src/executor.ts

```diff
@@ -72,6 +72,11 @@ const runServer = (cwd, serverPath, logger) => {
     logger('Starting backend');
 
     if (!nodeDebugOpt) {
+      if (!nodeDebugger) {
```

Comment:
You will still have `exec` called after if, which detects node version in order to find out proper debug option, allocates port and spawns server. Probably `exec` should be in `else` block after the `if`